apollo-runtime/src/main/java/com/apollographql/android/impl/ApolloClient.java

```diff
@@ -35,13 +35,13 @@
   private final Moshi moshi;
   private CallAdapter<R> adapter;
 
-  private ApolloClient(Builder<R> builder) {
+  private ApolloClient(Builder builder, CallAdapter<R> callAdapter) {
```

Comment:
so the call adapter stuff is going to go away after my pr #254 since we couldn't return a `R<T>` style thing